docs/03_tutorials/21_tic-tac-toe-game-smart-contract-Testnet.md

```diff
@@ -515,7 +515,7 @@ void tictactoe::move(const name &challenger, const name &host, const name &by, c
 }
 ```
 
-The complete tictactoe.cpp file can be downloaded from github here: [Tic-tac-toe tutorial cpp source.](https://github.com/EOSIO/welcome/tree/master/docs/03_tutorials/src/tictactoe/tictactoe.cpp "tic-tac-toe example cpp")
+The complete tictactoe.cpp file can be downloaded from github here: [Tic-tac-toe tutorial cpp source.](https://github.com/EOSIO/welcome/blob/release/2.0.x/src/tictactoe/tictactoe.cpp  "tic-tac-toe example cpp")
```

Comment:
Update or keep `release/2.0.x` if needed depending on which option (a,b,c) we select.